components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ScopeImpl.java

```diff
@@ -691,4 +716,49 @@ protected Element getImageDefinition(SVGDocument doc, String imgPath, int imgXLe
 
         return group;
     }
+
+    //Different Implementations for start and end scope icons
+    @Override
+    protected Element getEndImageDefinition(SVGDocument doc) {
+        return getStartEndImageDef(doc, getEndIconPath(), getEndIconXLeft(),
+                getEndIconYTop(), getEndIconWidth(), getEndIconHeight(),
+                getEndImageId());
+    }
+
+    protected Element getStartImageDefinition(SVGDocument doc) {
+        return getStartEndImageDef(doc, getStartIconPath(), getStartIconXLeft(),
+                getStartIconYTop(), getStartIconWidth(), getStartIconHeight(),
+                getStartImageId());
+    }
+
+    protected Element getStartEndImageDef(SVGDocument doc, String imgPath, int imgXLeft, int imgYTop,
+                                          int imgWidth, int imgHeight, String id) {
+
+        Element group = null;
+        group = doc.createElementNS("http://www.w3.org/2000/svg", "g");
```

Comment:
String "http://www.w3.org/2000/svg" has been used in several places. Create constants and reference where necessary.